.github/workflows/merge-gate.yml

```diff
@@ -106,7 +170,7 @@ jobs:
       contents: read
     # If you need additional jobs to be part of the merge gate, add them below
     # Removed saas-tests from being required
-    needs: [ fast-checks, large-runner-tests ]
+    needs: [ fast-checks, large-runner-tests, ordinary-itests ]
```

Comment:
As running GPU tests can be expensive, merging is possible without running GPU tests.